frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -1221,16 +1169,14 @@ export default ({
         // that'd require this entire function to be `async`, which could result
         // in race conditions as this is a watcher.
         Promise.resolve(sbp('chelonia/contract/isSyncing', toChatRoomId)).then((isSyncing) => {
-          // If the chatroom has changed since, return
-          if (this.summary.chatRoomID !== toChatRoomId) return
           if (isSyncing) {
-            toIsJoined && sbp('chelonia/queueInvocation', toChatRoomId, () => initAfterSynced(toChatRoomId))
+            sbp('chelonia/queueInvocation', initAfterSynced)
```

Comment:
`'chelonia/queueInvocation'` requires at least two arguments... I'm guessing this is missing the first argument (the `contractID` that represents the queue you're adding to).

Reply:
Adding to what @taoeffect said in the last paragraph, unless Chelonia is involved (i.e., using a `contractID` and requiring the behaviour above), `okTurtles.eventQueue/queueEvent` should be used.

Reply:
@taoeffect @corrideat These are helpful comments